cves/kernel/CVE-2021-38202.yml

```diff
@@ -148,10 +158,16 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    This issue appears to have been discovered on 2021-06-25.
+    There does not appear to be a written bug for this defect, and given that, 
+    the simplicity of the fix, the language in the commit message, and the 
+    repairing commit's proximity to other commits in the same file, it can be 
+    inferred that the mistake was discovered in passing while making a different
+    change, and was corrected on the spot.
+  automated: false
+  contest: false
+  developer: true
```

Comment:
You can find the developer name in the GitHub commit link on NVD

Reply:
Add the developer name instead of true

Reply:
"The automated, contest, and developer flags can be true, false, or nil."

the developer field is a boolean